src/Apps/Search/Routes/Artworks/Components/Filter/SearchResultsRefetch.tsx

```diff
@@ -54,6 +57,25 @@ export class SearchResultsRefetch extends Component<SearchRefetchProps> {
             console.error(error)
           }
 
+          const term = this.props.filtersState.keyword
+          const filters = Object.assign({}, this.props.filtersState)
+
+          const removeEmptyOrDefaults = obj => {
+            Object.keys(obj).forEach(
+              key => isDefaultFilter(key, obj[key]) && delete obj[key]
+            )
+          }
+          removeEmptyOrDefaults(filters)
+
+          const filterQueryParams = qs.stringify({
+            ...omit(filters, "keyword"),
+            term,
+          })
+
+          // TODO: Look into using router push w/ query params.
+          // this.props.router.replace(`/search2?${filterQueryParams}`)
```

Comment:
Since we're not using this, I can remove the `router` prop that's getting passed down here. But if there's no harm leaving it, might as well leave it if we think it's worth actually coming back to?

Reply:
Yeah, there's not a clear way forward ATM without a hard transition once using `router.push` (we don't want `replace`, then the users back / forward buttons wouldn't work). Might as well remove the `router` prop but leave the TODO. Can return once we get a sec.